### PR TITLE
[Whatsapp] handle document messages

### PR DESCRIPTION
**Pull request by AvinashReddy3108**
This patch enables matterbridge to relay Whatsapp's Document messages.
Tested it in my Whatsapp <-> Telegram bridge.

### CODE REVIEW

File: bridge/whatsapp/handlers.go [hunk 1]
```diff
@@ -312,3 +312,60 @@ func (b *Bwhatsapp) HandleAudioMessage(message whatsapp.AudioMessage) {
...
+		return
+	}
+
+	filename := fmt.Sprintf("%v%v", message.Info.Id, fileExt[0])
+
+	b.Log.Debugf("Trying to download %s with type %s", filename, message.Type)
+
+	data, err := message.Download()
+	if err != nil {
+		b.Log.Errorf("Download audio failed: %s", err)
```
Comment (42wim) [inline bridge/whatsapp/handlers.go:359]:
```suggestion
		b.Log.Errorf("Download document message failed: %s", err)
```

**Comment by 42wim** [comment]
Thanks! a small nitpick and then good to merge I think.

**Comment by AvinashReddy3108** [comment]
> Thanks! a small nitpick and then good to merge I think.

Merged the typo-fix you suggested, should be good to merge now.

**Comment by 42wim** [comment]
Thanks!

**Pull request merged**